doc/schemas/app_openapi.json

```diff
@@ -286,16 +286,6 @@
         "description": "Request payload must be a COSE Sign1 document, with expected protected headers. Signer must be a member identity registered with this service.",
         "scheme": "cose_sign1",
         "type": "http"
-      },
-      "member_signature": {
-        "description": "Request must be signed according to the HTTP Signature scheme. The signer must be a member identity registered with this service.",
-        "scheme": "signature",
-        "type": "http"
-      },
-      "user_signature": {
```

Comment:
Do we have an equivalent COSE signature scheme that users can use for apps, in both JS and C++?

Reply:
Not at the moment, no.